### ISSUE TITLE
can't save edited transaction?

### ISSUE THREAD

**Issue by rainforest1155** [arecker/bennedetto#69]
Not sure if this is a set up issue on my end, but if I try to edit a transaction, it fails to save. The console shows this:
`
[17/Feb/2016 01:23:35] "GET /api/reports/summary/ HTTP/1.1" 200 60
Not Found: /api/transactions/a43d2e80-0068-4758-9aa0-017ca86fef63
Internal Server Error: /api/transactions/a43d2e80-0068-4758-9aa0-017ca86fef63
Traceback (most recent call last):
  File "/home/user/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/home/user/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/middleware/common.py", line 114, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/home/user/.virtualenvs/bennedetto/local/lib/python2.7/site-packages/django/middleware/common.py", line 98, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via PUT, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining PUT data. Change your form to point to 192.168.188.2/api/transactions/a43d2e80-0068-4758-9aa0-017ca86fef63/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
[17/Feb/2016 01:23:48] "PUT /api/transactions/a43d2e80-0068-4758-9aa0-017ca86fef63 HTTP/1.1" 500 70603
`

I'm running Nginx on Debian Wheezy and routed Bennedetto through a reverse proxy with this config:
`
    location /  {
        proxy_pass      http://127.0.0.1:8000/;
        proxy_set_header        Host $host;
        proxy_set_header        X-Real-IP $remote_addr;
        proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_http_version      1.1;
    }
`

Currently, it's running via a separate server/port as I couldn't get the rewrite to work. However, I'd like to run it in a separate directory on my main server to be able to easily access it from outside my network. I'll figure this out some other time.

Any idea what might be the issue?

**Comment by arecker** [comment]
I think this might be a trailing slash issue.  I fixed it in the defunct `timezone-refactor` branch and may need to do the same in master.

``` javascript
     function ResourceFactory($resource, $http, APP_SETTINGS) {
          return {
              buildResource: function(endpoint) {
 -                var url = '{}{}/:id'.format(APP_SETTINGS.apiUrl, endpoint),
 +                var url = '{}{}/:id/'.format(APP_SETTINGS.apiUrl, endpoint),
                      params = {
                          id: '@id'
                      },
```

Basically, this needs to be done.

**Comment by rainforest1155** [comment]
Thanks for the pointer! I made the change and it seems to be working fine. Made a PR #73

**Comment by rainforest1155** [comment]
fixed with #75
